settings/eslint/eslintrc.js

```diff
@@ -0,0 +1,66 @@
+const commonParserOptions = {
+  ecmaFeatures: {
+    jsx: true,
+  },
+  ecmaVersion: '2020',
+  sourceType: 'module',
+};
+
+const commonRules = {
+  'prettier/prettier': [
+    'error',
+    {
+      singleQuote: true,
+      jsxSingleQuote: true,
+      trailingComma: 'all',
+    },
+  ],
+};
+
+module.exports = {
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
+  ignorePatterns: ['node_modules/', 'public/', 'build/', '!.*.js', '!.*.json'],
+  // Settings for .js, .jsx and .json
+  parser: 'babel-eslint',
```

Comment:
Maybe better to leave this as default

Reply:
Yeah, the default parser works fine. Cargo-culted from `eslint-plugin-json` :man_facepalming: